pubsub/pubsub.go

```diff
@@ -369,3 +372,22 @@ func defaultAckBatcher(ctx context.Context, s *Subscription) driver.Batcher {
 	}
 	return batcher.New(reflect.TypeOf([]driver.AckID{}).Elem(), maxHandlers, handler)
 }
+
+type errorCoder interface {
+	ErrorCode(error) gcerrors.ErrorCode
+}
+
+func wrapError(ec errorCoder, err error) error {
+	if err == nil {
```

Comment:
When would this happen? Seems like it's hiding a bug.

Reply:
`if err == nil` wasn't deleted?

Reply:
Ping on this one.

Reply:
Sorry, missed it. Done now.